src/pybind/mgr/telemetry/module.py

```diff
@@ -906,7 +909,39 @@ def compile_report(self, channels: Optional[List[str]] = None) -> Dict[str, Any]
                         'erasure_code_profile': ec_profile,
                         'cache_mode': pool['cache_mode'],
                     }
-                )
+
+                # basic_pool_usage collection (1/2)
+                if self.is_enabled_collection(Collection.basic_pool_usage):
+                    pool_data['application'] = []
+                    for application in pool['application_metadata']:
+                        # Only include default applications
+                        if application in ['cephfs', 'mgr', 'rbd', 'rgw']:
+                            pool_data['application'].append(application)
```

Comment:
I wrote this in a for loop to account for any extra keys that might appear in the 'application_metadata' dictionary. AFAIK, the dictionary will only ever contain one key, i.e. `pool['application_metadata'] = {'mgr': {...}}`, but I wanted to account for any scenarioes where there might be more than one key, just in case. i.e.  `pool['application_metadata'] = {'mgr': {...}, 'another_key': {...}}`.

Maybe this is unnecessary, but let me know what you prefer.

Reply:
it is likely that there is going to be only one key, but it's okay to leave it this way